src/main/java/com/helper/dto/response/AdminView.java

```diff
@@ -0,0 +1,48 @@
+package com.helper.dto.response;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AdminView {
+
+    Integer studentId;
+    String course_Name;
```

Comment:
wrong naming convention for course_Name it should be either courseName or course_name